packages/kbn-apm-types/src/es_fields/apm.ts

```diff
@@ -191,6 +191,9 @@ export const METRIC_OTEL_JVM_GC_DURATION = 'process.runtime.jvm.gc.duration';
 export const VALUE_OTEL_JVM_PROCESS_MEMORY_HEAP = 'heap';
 export const VALUE_OTEL_JVM_PROCESS_MEMORY_NON_HEAP = 'non_heap';
 
+export const TELEMETRY_SDK_NAME = 'telemetry.sdk.name';
```

Comment:
Nit: We can have a comment before those fields with the Otel doc linked (same as the description) to explain what those fields are. wdyt?

Reply:
done